sources/packages/backend/apps/api/src/services/disbursement-receipt/disbursement-receipt.service.ts

```diff
@@ -38,7 +40,12 @@ export class DisbursementReceiptService extends RecordDataModelService<Disbursem
         disbursementSchedule: {
           studentAssessment: {
             id: assessmentId,
-            application: studentId ? { student: { id: studentId } } : undefined,
+            application: {
+              id: applicationId ?? undefined,
```

Comment:
is `?? undefined` required here? if not present the value will automatically be undefined riht